rev-connect-api/src/main/resources/data.sql

```diff
@@ -0,0 +1,15 @@
+drop table if exists users cascade;
+
+create table users(
+    id BIGINT AUTO_INCREMENT PRIMARY KEY, 
+    username VARCHAR(255) NOT NULL UNIQUE 
```

Comment:
I think we need to add commas after every line so that this SQL statement is ran properly

Reply:
Okay